explorer/test/Test/Pos/Explorer/Socket/MethodsSpec.hs

```diff
@@ -50,7 +50,7 @@ import           Test.Pos.Explorer.MockFactory (mkTxOut)
 -- stack test cardano-sl-explorer --fast --test-arguments "-m Test.Pos.Explorer.Socket"
 
 spec :: Spec
-spec = beforeAll_ (setupLogging (defaultTestConfiguration Debug)) $
+spec = beforeAll_ (setupLogging "test" (defaultTestConfiguration Debug)) $
```

Comment:
`setupLogging "test" == setupTestLogging`?

Reply:
might be a shortcut; "test" is the dummy configuration key